src/client/src/OpenFin/apps/LimitChecker/LimitChecker.tsx

```diff
@@ -0,0 +1,36 @@
+import { Context, joinChannel } from "@finos/fdc3"
+import { useEffect } from "react"
+
+import { LimitCheckResultsTable } from "./LimitCheckResults"
+import { LimitInputs } from "./LimitInputs"
+import { checkLimit, LimitCheckerRequest } from "./state"
+
+const intentHandler = async (context: Context) => {
+  if (context.id) checkLimit(context.id as unknown as LimitCheckerRequest)
```

Comment:
any possibility of using [IntentResult](https://fdc3.finos.org/docs/api/spec#intent-results)

Reply:
just noticed - this is not an intent handler, is it?
if you could rename, would be most grateful

Reply:
and why is it `id`?